composer.json

```diff
@@ -24,8 +24,9 @@
         "php": "^8.0",
         "ext-pdo": "*",
         "psr/log": "^2.0|^3.0",
-        "psr/simple-cache": "^1.0.1",
+        "psr/simple-cache": "^2.0|^3.0",
         "yiisoft/db": "^3.0@dev",
+        "yiisoft/db-sqlite": "3.0.x-dev",
```

Comment:
Looks wrong. Concreate DB driver should not be in require.

Reply:
fix error typo done :)